tests/runner/CMakeLists.txt

```diff
@@ -31,7 +31,19 @@ macro (define_test_runner NAME)
         ${UACPI_INCLUDES}
     )
 
-    if (MSVC)
+    if (WATCOM)
```

Comment:
I think your commits got mixed up or something, because now the second commit only contains the cmakelists changes and random os.h changes, while the first commit enables CI which is not even runnable because these cmake changes are not yet in